lib/commentTreeUtils.es6.js

```diff
@@ -6,7 +6,7 @@ import { COMMENT_LOAD_MORE } from '../models2/thingTypes';
 export function treeifyComments(comments=[]) {
   const commentDict = {};
   comments.forEach(c => {
-    commentDict[c.uuid] = c;
```

Comment:
Is this another old bug? Guessing this was written assuming we got a model but it's really just `json` without the `uuid` set right?